framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -783,6 +786,9 @@ public void initChecker() {
         this.messagesProperties = getMessagesProperties();
 
         this.visitor = createSourceVisitor();
+
+        // Validate the lint flags.
+        this.activeLints = createActiveLints(getOptions());
```

Comment:
Without this change, executing `./checker/bin/javac -Alint=redundantNullComparison -processor nullness,regex  Repro.java` would not warn about an unknown lint flag, but running `./checker/bin/javac -Alint=redundantNullComparison -processor nullness,guieffect Repro.java` did produce a warning.
This now eagerly creates the active lint flags, always raising an error.
@smillst @mernst Does this sound good to you?

Reply:
This seems good.  Thanks.